File: backend/app/api/access_review.py
from fastapi import APIRouter

router = APIRouter()


@router.get("/access-review")
def get_access_review():

    return {
        "completedReviews": 284,
        "pendingReviews": 16,
        "overdueReviews": 5,
        "completionRate": 95,
        "nextCampaign": "Q3 Finance Review"
    }

File: frontend/src/types/accessReview.ts
export type GovernanceAction = {
  type: "overdue" | "pending";
  count: number;
  title: string;
  description: string;
};
export interface AccessReview {
  completedReviews: number;
  pendingReviews: number;
  overdueReviews: number;
  completionRate: number;
  nextCampaign: string;
}